snippets/rspec-mode/desct

```diff
@@ -0,0 +1,7 @@
+# -*- mode: snippet -*-
+# name: describe Class, "modifier" do ... end
```

Comment:
I think the `name` field needs to be updated.

Reply:
oh... I've fixed :sweat_smile: